examples/bond_simple.yml

```
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: network-test
+  vars:
+    network_connections:
+
+      # Create a bond master
+      - name: bond0
+        state: up
+        type: bond
+        interface_name: bond0
+        # bond configuration settings: (optional)
+        bond:
+          mode: active-backup
+          miimon: 110
+
+      # enslave an ethernet to the bond
+      - name: slave1
+        state: up
+        type: ethernet
+        interface_name: eth0
```

Please use eth1 and eth2 for the slave interfaces to make it easier to test the example in a VM.
```suggestion
        interface_name: eth1
```